src/csv2docx/csv2docx.py

```diff
@@ -30,17 +30,19 @@ def create_unique_name(filename: str, path: Path) -> Path:
     if filepath.exists():
         # Count available files with same name
         counter = len(list(path.glob(f"{filepath.stem}*docx"))) + 1
-        filepath = f"{path}/{filepath.stem}_{counter}.docx"
+        filepath = path / f"{filepath.stem}_{counter}.docx"
     return filepath
 
 
-def convert(data, template, name, path="output", delimiter=";"):
+def convert(
+    data: str, template: str, name: str, path: str = "output", delimiter: str = ";"
+) -> None:
     print("Getting .docx template and .csv data files ...")
 
     with open(data, "rt") as csvfile:
         csvdict = csv.DictReader(csvfile, delimiter=delimiter)
         csv_headers = csvdict.fieldnames
-        if name not in csv_headers:
+        if csv_headers and name not in csv_headers:
```

Comment:
I get the improvement here, but just curious - was this flagged by `mypy`?

Reply:
Yes, mypy flagged this as, which was solved by checking if `csv_headers` exists. The issue mypy raised was:

```
src/csv2docx/csv2docx.py:45: error: Unsupported right operand type for in ("Optional[Sequence[str]]")
```

I tried to reproduce this error by removing `csv_headers` and it no longer exists. This is because it is hidden by introducing `strict_optional = False` ([docs here](https://mypy.readthedocs.io/en/stable/config_file.html#none-and-optional-handling)) into mypy's init file, which I introduced afterwards to resolve another issue related to:

```python
column_in_data = set(docx_mergefields) - set(csv_headers)
```

mypy's output:

```
src/csv2docx/csv2docx.py:55: error: Argument 1 to "set" has incompatible type "Optional[Sequence[str]]"; expected "Iterable[str]"
src/csv2docx/csv2docx.py:55: error: Argument 1 to "set" has incompatible type "Optional[Sequence[str]]"; expected "Iterable[object]"
```

There was no elegant solution to address this -- [see mypy docs on the same issue of "invariance vs covariance"](https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance). I opted for the `strict_optional` as it seemed a good compromise.

Reply:
Got it, thanks for the elaborate explanation, helps me get a grip on the tools!